frontend/src/e2e-test/specs/0_citizen/citizen-holiday-reservations.spec.ts

```diff
@@ -46,27 +47,39 @@ beforeEach(async () => {
     .child(child1)
     .daycare(daycare)
     .with({
-      startDate: LocalDate.of(2035, 1, 1).formatIso(),
+      startDate: LocalDate.today().formatIso(),
```

Comment:
I believe the conclusion was that we want to avoid `LocalDate.today()` in tests and just use fixed time periods.